pkg/controllers/multicluster/serviceexport_controller.go

```diff
@@ -33,12 +34,18 @@ const (
 	EndpointSliceServiceLabel = "kubernetes.io/service-name"
 )
 
+var (
+	clusterId    string
+	clusterSetId string
```

Comment:
why is this required?

Reply:
Not required - but was an attempt to make it cleaner to access the ids throughout the controller without having to pass around the clusterId, clusterSetId I retrieve on lines 65 & 70. Another option is to call the getClusterId function again in extractEndpoints.

Reply:
> Another option is to call the getClusterId function again in extractEndpoints.

Let's do that. We do have context in the controller.

Reply:
done.